AK/UBSanitizer.h

```diff
@@ -20,6 +20,13 @@ class SourceLocation {
     u32 line() const { return m_line; }
     u32 column() const { return m_column; }
 
+    SourceLocation acquire()
```

Comment:
Is there a better name than aquire? I had to read the commit message to understand what was actually going on here...

Reply:
I suggest `permanently_clear()`

And before someone goes through the same thought process as I do: Yes, it actually *is* fine to permanently lose the data, because we currently have no way to switch `g_ubsan_is_deadly` to true again.

Reply:
Went with permanently_clear() and added a comment above the method